code/game/gamemodes/objective_items.dm

```diff
@@ -66,6 +71,7 @@
 	name = "a fire axe"
 	targetitem = /obj/item/fireaxe
 	excludefromjob = list(JOB_CHIEF_ENGINEER,JOB_STATION_ENGINEER,JOB_ATMOSPHERIC_TECHNICIAN)
+	item_owner = list(JOB_CHIEF_ENGINEER,JOB_STATION_ENGINEER,JOB_ATMOSPHERIC_TECHNICIAN)
```

Comment:
Does the fire axe also include the one on the Bridge (on most maps?). Could command staff care about the fire axe?

Reply:
Yes, I'm not all too sure what to do about that one to be honest because those are two very different locations so no list of owners is going to reliably cover both.
Maybe that's fine in this case on account of it being a bulky and difficult-to-conceal weapon.

Reply:
That did give me an idea though